connector.go

```diff
@@ -99,11 +100,11 @@ func NewConnector(dsn string) (*Connector, error) {
 	// resort is to use the current operating system provided user
 	// name.
 	if _, ok := o["user"]; !ok {
-		u, err := userCurrent()
```

Comment:
`userCurrent` is a local implementation to match the behavior of `libpq` on Windows:
https://github.com/lib/pq/blob/90697d60dd844d5ef6ff15135d0203f65d2f53b8/user_windows.go#L9-L17

If this PR works as expected in WebAssembly, then the better change is to add `js` here:
https://github.com/lib/pq/blob/d7918e36bee0ef49ef3dd9a17e575de83b26aac6/user_posix.go#L3

Reply:
Roger that, I'll close this PR then 👍